platform/macos/CHANGELOG.md

```diff
@@ -49,7 +49,7 @@
 
 ## 0.5.1
 
-This version of the Mapbox macOS SDK corresponds to version 3.6.4 of the Mapbox iOS SDK.
+This version of the Mapbox Maps SDK for macOS corresponds to version 3.6.4 of the Mapbox Maps SDK for iOS.
```

Comment:
These and subsequent references are to older versions of the iOS and macOS map SDKs that were called “Mapbox iOS SDK” and “Mapbox macOS SDK” (or “Mapbox OS X SDK”), respectively.